docs/user-guide/global-configurations/approval-policy.md

```diff
@@ -0,0 +1,136 @@
+# Approval Policy
+
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+When it comes to critical environments (let's say, production), you as a superadmin might want to introduce an approval flow for application deployment or changes made to the configuration files. Enforcing such restrictions will prevent unwanted deployments and direct modifications to sensitive configurations.
+
+The **Approval Policy** feature in Devtron lets you introduce an approval mechanism whenever your users perform the following actions:
+
+- Deploying an Application
```

Comment:
deploying an application doesn't seem right, its more of deploying an application to an environment type